docs/source/release_notes/release-notes.rst

```diff
@@ -59,16 +75,12 @@ develop
          - Upgraded Postgres jdbc driver to 42.2.1 (from 9.4.1209).
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2985>`__)
 
-    *    - |improved|
-         - Wrap remote calls from AtlasDB with http-remoting tracing.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/2986>`__)
-
     *    - |fixed|
          - Fix NPE when warming conflict detection cache if table is being created.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2993>`__)
 
     *    - |improved|
-         - The ``writeThreshold`` and ``writeSizeThreshold`` parameters for when to write stats for the Sweep prioritization.
+         - Introduced ``writeThreshold`` and ``writeSizeThreshold`` parameters for when to write stats for the Sweep prioritization.
```

Comment:
Maybe "Introduced configurable"? Otherwise it sounds like it might be an internal code detail.